developer-docs-site/docs/guides/install-petra-wallet.md

```diff
@@ -5,24 +5,38 @@ slug: "install-petra-wallet-extension"
 
 # Install Petra Extension
 
-Petra (Aptos Wallet) is now available on the Chrome store. To install the Petra extension on your Chrome browser, you can chose either of the following options:
+[Petra (Aptos Wallet)](https://petra.app/) is now available on the Chrome store. To install the Petra extension on your Chrome browser, you can chose either of the following options:
 
 ## Install from the Chrome Web Store
 
+For the latest stable release, install the extension from the Chrome Web Store:
+
 1. Visit the [Petra Wallet extension page](https://chrome.google.com/webstore/detail/petra/ejjladinnckdgjemekebdpeokbikhfci).
-2. Click the **Add to Chrome** button.
+1. Click the **Add to Chrome** button.
+1. Find your Petra Wallet in your Chrome extensions anytime by loading this in the browser location (URL) bar: chrome://extensions/
+1. Optionally, [pin the extension](https://support.google.com/chrome/a/answer/11190170) to make it findable by Petra button in your Chrome browser toolbar.
```

Comment:
to make it findable by Petra button in your Chrome browser toolbar

to

to make it findable in your Chrome browser toolbar